docs/yarn.lock

```diff
@@ -4,45 +4,45 @@
 
 "@algolia/autocomplete-core@1.7.2":
   version "1.7.2"
-  resolved "https://registry.yarnpkg.com/@algolia/autocomplete-core/-/autocomplete-core-1.7.2.tgz#8abbed88082f611997538760dffcb43b33b1fd1d"
+  resolved "https://registry.npmjs.org/@algolia/autocomplete-core/-/autocomplete-core-1.7.2.tgz"
```

Comment:
Curious why the registry changed here. We may want to run `yarn` again. Thanks

Reply:
@jamalavedra let's fix this